src/platform/windows/window.rs

```diff
@@ -1019,13 +1023,22 @@ unsafe fn init(
     // making the window transparent
     if attributes.transparent && !pl_attribs.no_redirection_bitmap {
         let bb = dwmapi::DWM_BLURBEHIND {
-            dwFlags: 0x1, // FIXME: DWM_BB_ENABLE;
+            dwFlags: dwmapi::DWM_BB_ENABLE | dwmapi::DWM_BB_BLURREGION,
```

Comment:
These constants appear to be absent from winapi 0.3.5. It would be great if you'd change Cargo.toml to use winapi 0.3.6, since then people won't have to manually run `cargo update`.